yt/data_objects/tests/test_fluxes.py

```diff
@@ -115,10 +109,18 @@ def test_correct_output_unit():
     sur = ds.surface(sp1,"HI_Density", .5*Nmax)
     sur['x'][0]
```

Comment:
you can delete this original test now i think

Reply:
done